test/controllers/CheckDetailsControllerSpec.scala

```diff
@@ -306,6 +306,30 @@ class CheckDetailsControllerSpec extends SpecBase with SummaryListFluency {
         }
       }
 
+      "when PDV data not found" in {
+        val app = applicationBuilder(userAnswers = Some(emptyUserAnswers))
+          .overrides(
+            inject.bind[PersonalDetailsValidationService].toInstance(mockPersonalDetailsValidationService),
+            inject.bind[CheckDetailsService].toInstance(mockCheckDetailsService),
+            inject.bind[AuditService].toInstance(auditService)
+          )
+          .build()
+
+        when(mockPersonalDetailsValidationService.createPDVDataFromPDVMatch(any())(any()))
+          .thenReturn(Future.successful(mockPDVResponseDataSuccess))
```

Comment:
Is this returning a not found? Reads like a success response?

Reply:
oh right, I fixed it. now